### ISSUE TITLE
Resolve ROW Portal Accessibility Common Issues

### ISSUE THREAD

**Issue by ChrispinP** [cityofaustin/atd-data-tech#27231]
- [x] Container Element is Empty : remove empty ul tags
- [x] Email Link Contrast : similar but darker blue color for 5.0 contrast which meets 4.5 minimum
- [x] Email Link Alt Text : add aria-label for screen reader

**Comment by ChrispinP** [comment]
completed in prod. refreshed siteimprove and now shows these A and AA issues as resolved. Only remaining issue now for ROW Portal is the restricted page zoom.

**Comment by ChrispinP** [comment]
Completed second pass at accessiblity items for the ROW Portal according to our DTS inventory and using the SiteImprove extension.

Added alt text for Account Signup Confirmation image. SiteImprove thinks all close buttons should have alt text, the green confirm button should have better contrast which would require a redesign, and is confused about mutl-choice/connection select fields where it either thinks the Select text is invisible, there is no label, or the value is getting clipped, none of which are the case. It also mistakenly thinks an empty h2 heading is present when it is not.

**Comment by ChrispinP** [comment]
Any further updates from systemic changes or re-designs for accessibility will be captured in [27690](https://github.com/cityofaustin/atd-data-tech/issues/27690)

All common issues have been reviewed/resolved.
